packages/inter-protocol/test/swingsetTests/governance/vat-alice.js

```diff
@@ -48,6 +48,7 @@ const build = async (log, zoe, brands, payments, timer) => {
     );
 
     const assetNotifier = makeNotifierFromSubscriber(
+      // @ts-expect-error legit error broken in refactoring
```

Comment:
Maybe mark with a `FIXME` if it's something to be fixed.

Reply:
this is in a dead test we keep around in case we ever want this style of test again. so normally I would put a FIXME but I don't want this to come up in a search for outstanding things to fix; it can stay broken.

incidentally, @Chris-Hibbert WDYT of deleting these swingset tests at this point? They're happy path tests that get stressed in other ways now.

Reply:
What other test covers governance of vault parameters and the interaction with charging interest?

Vault governed parameters are different from other contracts, since much of the boilerplate that is shared among other contracts has to be handled ideosyncratically for vaults, since there are sub-paramManagers for each vaultManager.  (i.e. Vaults don't use contractHelper.)